assets/charts/components/azure-arc-lokomotive/templates/remove.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-delete"
+  name: azure-arc-remove
+  namespace: {{ .Values.namespace }}
```

Comment:
We don't need namespace here? And if so, we could use `{{ .Release.Namespace  }}` ?